br/pkg/lightning/restore/check_info_test.go

```diff
@@ -484,15 +483,16 @@ func (s *checkInfoSuite) TestCheckTableEmpty(c *C) {
 		WillReturnRows(sqlmock.NewRows([]string{""}).AddRow(1))
 	rc.checkTemplate = NewSimpleTemplate()
 	err = rc.checkTableEmpty(ctx)
-	c.Assert(err, IsNil)
-	c.Assert(mock.ExpectationsWereMet(), IsNil)
+	require.NoError(t, err)
+	require.NoError(t, mock.ExpectationsWereMet())
+
 	tmpl := rc.checkTemplate.(*SimpleTemplate)
-	c.Assert(len(tmpl.criticalMsgs), Equals, 1)
-	c.Assert(tmpl.criticalMsgs[0], Matches, "table\\(s\\) \\[`test2`.`tbl1`\\] are not empty")
+	require.Equal(t, 1, len(tmpl.criticalMsgs))
+	require.Regexp(t, "table\\(s\\) \\[`test2`.`tbl1`\\] are not empty", tmpl.criticalMsgs[0])
```

Comment:
```suggestion
	require.Equal(t, "table(s) [`test2`.`tbl1`] are not empty", tmpl.criticalMsgs[0])
```

I guess using `Matches` is a mistake :P